### ISSUE TITLE
The sprite is not saved when building the project

### ISSUE THREAD

**Issue by ghost** [rodshtein/sveltekit-sprite#3]

**Comment by ghost** [comment]
When running the "vite build" command, the plugin generates a sprite in the template, but after completion and when running the script, the sprite is not added to the template and it does not exist at all

**Comment by ghost** [comment]
xd @rodshtein

**Comment by ghost** [comment]
Я извиняюсь, так вы ещё и русскоговорящий, так будет проще)

**Comment by rodshtein** [comment]
Это поведение воспроизводится на чистой сборке?

**Comment by ghost** [comment]
Да. При dev сборке на vite, всё отрабатывает как надо. Когда отключаешь dev сборку возвращается тег на место спрайта, всё как надо. А при билде, спрайт не сохраняется.

**Comment by ghost** [comment]
Единственный вариант какой я нашёл это просто во время разработки скопировать тег спрайта, и перед билдом вставить за место тега плагина.

**Comment by ghost** [comment]
Единственное что не из чистой сборки - это adapter-node, но без него не происходит билд проекта

**Comment by ghost** [comment]
Добрый день. Существует ли решение данной проблемы?

**Comment by rodshtein** [comment]
Дело в том, что раньше всё работало. Возможно что-то поменялось в vite. Мне нужно чуть времени на проверку.
